etc/50-default.conf

```diff
@@ -0,0 +1,70 @@
+#  Default rules for rsyslog.
```

Comment:
what this file is about?

Reply:
to remove all the unwanted syslog.  today we pick and edit lines.  Copying the file over is more efficient. I have added removal of the console stuff